README.md

```diff
@@ -22,6 +22,17 @@ From the project root, run the following commands:
 1. `yarn install`
 1. `yarn storybook`
 
+## Web Components
+
+The Design System team is working on replacing the React components in this library with Web Component versions. This isn't _just_ a React replacement however - part of the goal is to have true components that can be used outside of React in plain HTML pages.
+
+For now these Web Components live on the [`web-components` branch](https://github.com/department-of-veterans-affairs/component-library/tree/web-components). In order to add a new component or make changes to an existing one, you must:
+
+1. Create a new branch off of `web-components`
+1. Have the PR be based off of `web-components` (not `master`)
```

Comment:
Bonus points for a link or screenshot explaining what this means. I can see somebody getting confused. ("Well yes, I branched off `web-components`, so the PR is based on that branch...?" But this is specifically talking about the _merge_ base.)